fastdtw/fastdtw.py

```diff
@@ -85,7 +85,10 @@ def __prep_inputs(x, y, dist):
         raise ValueError('dist cannot be a negative integer')
 
     if dist is None:
-        dist = __difference
+        if x.ndim == 1:
+            dist = __difference
+        else: 
+            dist = __norm(p=1)
```

Comment:
Would it be possible to figure out the user-provided dist is `None` in the procedure entry point and just set it to something reasonable (i.e. manhattan) there? That way, I think, we could avoid all `is None` checks later on in the core of the procedure.